.github/workflows/ci.yml

```diff
@@ -149,16 +162,25 @@ jobs:
       VS_GENERATOR_TOOLSET: ${{ matrix.VS_GENERATOR_TOOLSET }}
 
     steps:
+      - name: Installing Alpine Linux Dependencies
+        if: ${{ contains(matrix.container, 'alpine') }}
+        run: |
+          apk update
+          apk add bash build-base cargo cmake curl curl-dev git icu linux-headers mitmproxy perl python3-dev sudo ${{ matrix.APK_PACKAGES }}
```

Comment:
Note: git needs to be installed before actions/checkout

Reply:
ditto: https://github.com/getsentry/sentry-native/pull/1233#discussion_r2086210671